py/seapy

```diff
@@ -44,10 +43,9 @@ def main (argv):
             sea.commands.Inspect,
             sea.commands.SimpleMemoryChecks(),
             sea.commands.Smc,
-            sea.commands.SeaExeCex()
+            sea.commands.SeaExeCex(),
```

Comment:
it's a python thing. A list can be terminated by a comma. This way when you add a new item, you don't need to change the previous one.

Reply:
@edmcman: recently, we are trying to work on the `deep-dev` branch  because it will become master hopefully soon. Do you mind to make a new PR in the `deep-dev` branch? This also means that it would be great if you can switch to that branch. This branch uses the latest version of spacer. @agurfinkel: are you OK with that?

Reply:
👍

Reply:
The comma thing was an accident :)  I added a new command and then removed it.  But @agurfinkel is right... python doesn't care.

I'll close this PR and send a new one soon for deep-dev.

Reply:
:) I prefer a comma, I thought you did it on purpose. I certainly would have